workspaces/liferay-partner-workspace/client-extensions/liferay-partner-custom-element/src/routes/PartnerOpportunitiesList/utils/constants/getCloseDateFilterTerm.ts

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {INITIAL_FILTER} from './initialFilter';
+
+export default function getCloseDateFilterTerm(
+	initialFilter: string,
+	activityPeriod: typeof INITIAL_FILTER.closeDate
+) {
+	const filterDates = [];
+
+	if (activityPeriod.dates.startDate) {
+		filterDates.push(`closeDate ge ${activityPeriod.dates.startDate}`);
+	}
+	if (activityPeriod.dates.endDate) {
```

Comment:
there should be an extra line break to separate these two if blocks

Reply:
Done!